test/common/README.md

```diff
@@ -413,6 +413,19 @@ Platform normalizes the `pwd` command.
 
 Synchronous version of `spawnPwd`.
 
+## ArrayStream Module
```

Comment:
Does arrayStream needs to be removed from line 41?

Reply:
Since that seemed self-evident, I went ahead and did it. Separate commit so if I'm wrong, it can be rebased out easily.